### PR TITLE
Add pbipa-0.0.1

### PR DESCRIPTION
**Pull request by pb-cdunn**
New package: pbipa (PacBio Improved Phased Assembler)

### CODE REVIEW

**Comment by pb-cdunn** [comment]
```
19:47:13 BIOCONDA INFO (OUT) Collecting package metadata: ...working... done
...19:52:08 BIOCONDA INFO (OUT) Solving environment: ...working... /opt/build_script.bash: line 21:    33 Killed                  conda build --override-channels --no-anaconda-upload -c conda-forge -c bioconda -c defaults -e /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml -e /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml /opt/recipe/meta.yaml 2>&1
19:52:10 BIOCONDA ERROR COMMAND FAILED (exited with 137): docker run -t --net host --rm -v /tmp/tmp7nrdpnvk/build_script.bash:/opt/build_script.bash -v /home/circleci/project/miniconda/conda-bld/:/opt/host-conda-bld -v /home/circleci/project/recipes/pbipa:/opt/recipe -e LANG=en_US.UTF-8 -e HOST_USER_ID=1001 bioconda/bioconda-utils-build-env:latest /bin/bash /opt/build_script.bash

19:52:10 BIOCONDA ERROR BUILD FAILED recipes/pbipa
```

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/pbipa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+extra:
+  recipe-maintainers:
+    - pb-cdunn
+  skip-lints:
+    - should_be_noarch_generic # We use "skip", so this is a flawed lint-check.
+
+build:
+  number: 0
+  skip: True # [py27 or py36 or osx]
```
Comment (dpryan79) [inline recipes/pbipa/meta.yaml:21]: Please change this to `skip: True  # [osx]`

File: recipes/pbipa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
```
Comment (dpryan79) [inline recipes/pbipa/meta.yaml:24]: Please remove pcre and python. Also please add the following:

    build:
      - {{ compiler('c') }}

Then you can omit the skip-lints

**Comment by pb-cdunn** [comment]
@dpryan79 , I incorporated your suggestions. Thanks.

The problem is with the addition of `snakemake` as a requirement. (I.e. last commit failed, but next to last passed.) How can I do that?

**Comment by pb-cdunn** [comment]
Getting closer! **snakemake-minimal** works, or maybe we had to drop `python >=3.5`. But we got a passing build. Now I'm adding the runtime dependencies ...

**Comment by dpryan79** [comment]
Looks like pbmm2 needs to be rebuilt, we just updated the htslib pinnings.

**Comment by pb-cdunn** [comment]
Unfortunately, we (at PacBio) are unable to switch to htslib-1.10 at this time. (I'm told this relates to `pysam`, so I'm surprised you have been able to update **htslib** in Bioconda.)

For now, pinning to htslib-1.9 seems to work.

**Comment by armintoepfer** [comment]
The build is green now. Can we move forward with sticking to the pinned htslib 1.9 version until we had time to upgrade pbbam and pbmm2 to the latest htslib? Htslib changed behaviour in some relevant cases that need to be addressed first. But that shouldn't block this PR here.

**Comment by armintoepfer** [comment]
I'm upgrading pbbam and pbmm2 to build against latest htslib

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by pb-cdunn** [comment]
Thank you! It's working great.

**Pull request merged**